awscli/customizations/configure.py

```diff
@@ -302,6 +303,9 @@ def _run_main(self, args, parsed_globals):
         region = self._lookup_config('region')
         self._display_config_value(region, 'region')
 
+        ca_bundle = self._lookup_config('ca_bundle')
```

Comment:
I'd prefer to remove this from the `list` command for now as well.

`aws configure/aws configure list` currently show the bare minimum configuration values that a customer needs to get started.

I'd like to approach this in a more general manner and have something like `aws configure list --all` to list all the known CLI options.  There's other config values that `aws configure list` is not currently showing so we probably need an option that shows everything.